lib/srv/usermgmt_unix.go

```diff
@@ -0,0 +1,95 @@
+//go:build !windows
+// +build !windows
```

Comment:
`_unix.go` already implicitly has a tag filter for `unix`, doesn't it?

Reply:
changed to _linux